### ISSUE TITLE
Update documentation on metrics

### ISSUE THREAD

**Issue by KonstantinaLazaridou** [autonomio/talos#350]
Could you please update the documentation https://autonomio.github.io/docs_talos/#reporting regarding the `_acc` suffixes in the metric names?

For instance:
The documentation says we should use `r.high('val_fmeasure')`,
but this is the correct usage as far as I understand:
`model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['acc', fmeasure_acc]);`
`print("Highest val_fmeasure_acc: {}".format(report.high(metric="val_fmeasure_acc")));`

Thanks!

**Comment by mikkokotila** [comment]
Thanks. The documentation will have to be majorly changed/improved before the new LTS i.e. >=v.0.6 goes to master, which should be soon. There is a bunch of tickets related with documentation, and all will be handled at that time.

**Comment by mikkokotila** [comment]
This is now fixed in the new upcoming docs. Already available in `/docs` in daily-dev. Closing here.
